src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -153,209 +184,241 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, route}: Workspac
 
     /** Call update workspace currency and hide the modal */
     const confirmCurrencyChangeAndHideModal = useCallback(() => {
-        Policy.updateGeneralSettings(policyID, policyName, CONST.CURRENCY.USD);
+        updateGeneralSettings(policyID, policyName, CONST.CURRENCY.USD);
         setIsCurrencyModalOpen(false);
-        ReimbursementAccount.navigateToBankAccountRoute(policyID);
+        navigateToBankAccountRoute(policyID);
     }, [policyID, policyName]);
 
-    const hasMembersError = PolicyUtils.shouldShowEmployeeListError(policy);
-    const hasPolicyCategoryError = PolicyUtils.hasPolicyCategoriesError(policyCategories);
+    const hasMembersError = shouldShowEmployeeListError(policy);
+    const hasPolicyCategoryError = hasPolicyCategoriesError(policyCategories);
     const hasGeneralSettingsError =
         !isEmptyObject(policy?.errorFields?.name ?? {}) ||
         !isEmptyObject(policy?.errorFields?.avatarURL ?? {}) ||
         !isEmptyObject(policy?.errorFields?.ouputCurrency ?? {}) ||
         !isEmptyObject(policy?.errorFields?.address ?? {});
-    const shouldShowProtectedItems = PolicyUtils.isPolicyAdmin(policy, login);
-    const isPaidGroupPolicy = PolicyUtils.isPaidGroupPolicy(policy);
+    const shouldShowProtectedItems = isPolicyAdmin(policy, login);
+    const isPaidGroupPolicyValue = isPaidGroupPolicy(policy);
     const [featureStates, setFeatureStates] = useState(policyFeatureStates);
+    const [highlightedFeature, setHighlightedFeature] = useState<string | undefined>(undefined);
+
+    const workspaceMenuItems: WorkspaceMenuItem[] = useMemo(() => {
```

Comment:
@mountiny 
>@sumo-slonik could you please highligh in the diff the changes made to fix the issue found in staging? thanks!

The addition of mnemonization here eliminates the problem that caused unnecessary rerendering and caused the flag responsible for highlighting to be removed.